dev/ast/cross_join.h

```diff
@@ -0,0 +1,31 @@
+#include "../functional/config.h"
```

Comment:
Please remove this include. It would be very cumbersome to include it everywhere, so it's added at the very beginning of the amalgamation header so it is universally available.